src/api/starship.js

```diff
@@ -26,3 +27,33 @@ starship.fetch = id => {
     }, reject);
   });
 };
+starship.getPicture = searchWord => {
```

Comment:
This picture search appears to be the exact same for each of the component API files. Would prefer moving this to a picture API file and delegating to that.